RELEASE-NOTES.md

```diff
@@ -1,98 +1,106 @@
-* `snow snowpark registry` was replaced with `snow registry` command.
-* `snow connection test` now outputs all connection details (except for the password), along with connection status
-* Removed `snow snowpark function create` and `snow snowpark function update`. Procedures can be deployed using `snow snowpark function deploy`.
-* Removed `snow snowpark procedure create` and `snow snowpark procedure update`. Procedures can be deployed using `snow snowpark procedure deploy`.
-* From `snow streamlit deploy` moved following parameters to `snowflake.yml`:
-  * Argument:
-    * `streamlit-name`
-  * Flags:
-    * `--file`
-    * `--stage`
-    * `--env-file`
-    * `--pages-dir`
-* `init` commands for functions and procedures create new project in new directory instead of using current working directory.
-* Moved `snow stage` from top-level to `snow object` subgroup
-* `snow warehouse status` is now `snow object show warehouse`
-* Introduced `snow object` group with `show`, `describe` and `drop` commands for: compute pools,
-databases, tables, warehouses, functions, procedures, roles, schemas, services, jobs and streamlits
 
+- Removed `create` and `update` commands for snowpark functions and procedures. Please use `deploy` command.
+- Changed path to coverage reports on stage, previously created procedures with coverage will not work, have to be recreated.
+- Update function or procedure will upload function/procedure code to new path on stage. Previous code will remain under old path on stage.
+- Functions and procedures use single zip artifact for all functions and procedures.
+- Snowpark Containers services commands
+  - `compute-pool` commands and its alias `cp` were renamed to `pool` commands.
+  - `jobs` commands were renamed to `job`.
+  - `services` commands were renamed to `service`
+  - `pool`, `job` and `service` commands were moved from `snowpark` group to a new `containers` group.
+- `snow snowpark registry` was replaced with `snow registry` command.
+- `snow connection test` now outputs all connection details (except for the password), along with connection status
+- Removed `snow snowpark function create` and `snow snowpark function update`. Procedures can be deployed using `snow snowpark function deploy`.
+- Removed `snow snowpark procedure create` and `snow snowpark procedure update`. Procedures can be deployed using `snow snowpark procedure deploy`.
+- From `snow streamlit deploy` moved following parameters to `snowflake.yml`:
+  - Argument:
+    - `streamlit-name`
+  - Flags:
+    - `--file`
+    - `--stage`
+    - `--env-file`
+    - `--pages-dir`
+- `init` commands for functions and procedures create new project in new directory instead of using current working directory.
+- Moved `snow stage` from top-level to `snow object` subgroup
+- `snow warehouse status` is now `snow object show warehouse`
+- Introduced `snow object` group with `show`, `describe` and `drop` commands for: compute pools,
```

Comment:
I'm not sure this is good idea. Because git blame will show different commit for each line than the original one. In case of realase notes it's quite useful.

Reply:
Sorry about that -- I have an auto-formatter for markdown. I'll undo this.